_posts/2023-04-07-model-registry.md

```diff
@@ -0,0 +1,171 @@
+┠────────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┨
+┃ METADATA               │ {'zenml_version': '0.34.0', 'zenml_run_name': 'mlflow_training_pipeline-2023_03_01-08_08_52_398499', 'zenml_pipeline_name': 'mlflow_training_pipeline', 'zenml_pipeline_run_uuid': '29fb22c1-6e0b-4431-9e04-226226506d16',              ┃
+┃                        │ 'zenml_workspace': '10e060b3-2f7e-463d-9ec8-3a211ef4e1f6', 'lr': '0.001', 'epochs': '5', 'optimizer': 'Adam'}                                                                                                                                  ┃
+┠────────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┨
+┃ MODEL_SOURCE_URI       │ file:///Users/safoine-zenml/Library/Application Support/zenml/local_stores/0902a511-117d-4152-a098-b2f1124c4493/mlruns/489728212459131640/293a0d2e71e046999f77a79639f6eac2/artifacts/model                                                     ┃
+┠────────────────────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┨
+┃ STAGE                  │ None                                                                                                                                                                                                                                           ┃
+┗━━━━━━━━━━━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+```
+
+Finally to delete a registered model or a specific model version, 
+you can use the `zenml model-registry models delete REGISTERED_MODEL_NAME` and 
+`zenml model-registry models delete-version REGISTERED_MODEL_NAME -v VERSION` 
+commands respectively.
+
+## Conclusion
+
+Incorporating a model registry into your ZenML MLOps stack offers numerous benefits. It can speed up deployment, simplify model lifecycle management, enhance collaboration, and improve model governance. By integrating the MLflow Model Registry with ZenML, you can further streamline the management and deployment of your machine learning models. Embrace the power of model registries in your machine learning workflows and take advantage of their comprehensive capabilities.
```

Comment:
You should include a link to the docs for model registry, also point people to the example where they can get going with it, and then also include a link to slack for if people have questions.